storage/addressing.go

```diff
@@ -62,6 +62,13 @@ func updateRangeAddressing(b *client.Batch, desc *roachpb.RangeDescriptor) error
 	return rangeAddressing(b, desc, putMeta)
 }
 
+func makeMetaKey(prefix roachpb.Key, key roachpb.RKey) roachpb.Key {
```

Comment:
Probably. I was doing the minimal change here. Let me take a look at moving it to keys.